pkg/eth/erc20/bridge.go

```diff
@@ -0,0 +1,213 @@
+package erc20
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"path/filepath"
+	"sync"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	ethtypes "github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/threefoldfoundation/tfchain/pkg/persist"
+	tfchaintypes "github.com/threefoldfoundation/tfchain/pkg/types"
+	"github.com/threefoldtech/rivine/modules"
+	"github.com/threefoldtech/rivine/types"
+)
+
+const (
+	// blockDelay is the amount of blocks to wait before
+	// pushing tft transactions to the ethereum contract,
+	// and to push ethereum transactions to the TF chain
+	blockDelay = 6
```

Comment:
@LeeSmet can we make this oublic? Than it also generates in the Godocs, as it is a constant, I do not see why it cannot be public anyhow.